testutils/infra/cli.py

```diff
@@ -121,15 +125,28 @@ def migrate(self):
 
 class CliDeviceauth(BaseCli):
     def __init__(self, containers_namespace="backend-tests", container_manager=None):
+        """ Instantiate deviceauth microservice CLI class. Both open source and enterprise versions are supported. """
         BaseCli.__init__(
             self, "mender-device-auth", containers_namespace, container_manager
         )
 
+        open_source = Service("/usr/bin/deviceauth", "/etc/deviceauth")
+        enterprise = Service(
+            "/usr/bin/deviceauth-enterprise", "/etc/deviceauth-enterprise"
+        )
+
+        for service in [open_source, enterprise]:
+            try:
+                self.container_manager.execute(self.cid, [service.bin_path, "--help"])
+                self.service = service
+            except:
+                continue
```

Comment:
Looks good, but to make we have found at least one valid binary:
```suggestion
        for service in [open_source, enterprise]:
            try:
                self.container_manager.execute(self.cid, [service.bin_path, "--help"])
                self.service = service
                break
            except:
                continue
        else:
            raise RuntimeError("no runnable binary found in mender-deviceauth")
```

Reply:
Alternatively, as done in other services, something like:
```
        if self.service is None:
            raise RuntimeError("no runnable binary found in mender-useradm")
```

But I personally prefer the former.

Reply:
Added RuntimeError, thanks!